setEnvironmentVariables.ps1

```diff
@@ -39,3 +39,7 @@
 # If needed (to execute CI Tests locally) - set to Value that can be found in internal wiki [Environment]::SetEnvironmentVariable("KEYCLOAK_LINKEDIN_SECRET", "#", [System.EnvironmentVariableTarget]::User)
 [Environment]::SetEnvironmentVariable("PROXY_PRIMARY_URL", "local-dev.dataland.com", [System.EnvironmentVariableTarget]::User)
 [Environment]::SetEnvironmentVariable("PROXY_LETSENCRYPT_PATH", "/etc/letsencrypt/local-dev.dataland.com", [System.EnvironmentVariableTarget]::User)
+[Environment]::SetEnvironmentVariable("RABBITMQ_USER", "admin", [System.EnvironmentVariableTarget]::User)
+[Environment]::SetEnvironmentVariable("RABBITMQ_PASS", "dataland", [System.EnvironmentVariableTarget]::User)
```

Comment:
We now have - in the local environment - 3 default passwords: 
password, test and dataland.

Maybe we should streamline this?

Reply:
WIP

Reply:
Don't forget "admin".

Reply:
ok